src/test/rustdoc/source-version-separator.rs

```diff
@@ -0,0 +1,31 @@
+#![stable(feature = "bar", since = "1.0")]
+#![crate_name = "foo"]
+
+#![feature(staged_api)]
+
+// @has foo/trait.Bar.html
+// @has - '//div[@class="main-heading"]/*[@class="out-of-band"]' '1.0· source · '
+#[stable(feature = "bar", since = "1.0")]
+pub trait Bar {
+    // @has - '//div[@id="tymethod.foo"]/*[@class="rightside"]' '3.0 · source'
+    #[stable(feature = "foobar", since = "3.0")]
+    fn foo();
+}
+
+// @has - '//div[@id="implementors-list"]//*[@class="rightside"]' '4.0 · source'
+
+// @has foo/struct.Foo.html
+// @has - '//div[@class="main-heading"]/*[@class="out-of-band"]' '1.0· source · '
```

Comment:
There's a missing space here.

```suggestion
// @has - '//div[@class="main-heading"]/*[@class="out-of-band"]' '1.0 · source · '
```

Reply:
No, there isn't (otherwise the test would fail). :)

Reply:
With an image you'll understand better what's happening:

![Screenshot from 2022-01-16 17-46-31](https://user-images.githubusercontent.com/3050060/149669350-8966526c-a92f-4f33-8915-eef7506fb953.png)

Reply:
Ah, got it. It looks like there's a missing-space problem in the templates, but it's papered over by padding. I'll fix in a followup PR.